### ISSUE TITLE
We really ought to support contains(Matcher<T> T) et al.

### ISSUE THREAD

**Issue by CliveEvans** [unruly/java-8-matchers#10]
I honestly hadn't noticed we don't support this before:

```java
assertThat(
    Stream.of("one", "two", "three"),
    StreamMatchers.contains(equalTo("one"), equalTo("two"), equalTo("three"))
);
```

**Comment by bruceeddy** [comment]
Good spot, Clive.  We'll take a look.

**Comment by oniekrenz** [comment]
It's been two years now. Is this project dead? Because I would like to have this feature too.

**Comment by mrwilson** [comment]
@oniekrenz - this is still a maintained project. I've opened a pull request with an implementation https://github.com/unruly/java-8-matchers/pull/14

**Comment by mrwilson** [comment]
@oniekrenz The change above has been merged and we'll push this to Maven Central in the next day or two. Thanks!

**Comment by mrwilson** [comment]
This has been released on Maven central as version 1.6 - enjoy!
